caffe2/python/onnx/backend.py

```diff
@@ -381,12 +381,13 @@ def _make_rnn_direction(cls, input_blob, B, W, R, initial_states_and_names, sequ
     @classmethod
     def _create_upsample(cls, init_model, pred_model, n, opset_version):
         c2_op = cls._common_onnx_node_to_caffe2_op(init_model, pred_model, n, opset_version)
-        if len(n.attrs['scales']) != 4:
-            raise ValueError("The scales argument should have size 4")
-        elif not (np.isclose(n.attrs['scales'][0], 1) and np.isclose(n.attrs['scales'][1], 1)):
-            raise ValueError("The first two elements in the scales argument must be 1")
-        c2_op.arg.extend([caffe2.python.utils.MakeArgument('height_scale', n.attrs['scales'][2])])
-        c2_op.arg.extend([caffe2.python.utils.MakeArgument('width_scale', n.attrs['scales'][3])])
+        if opset_version >= 7:
```

Comment:
When can we have one version of the converter as we did for the c2->onnx exporter? :)

Reply:
Good question, let's do it! How about a bootcamp task?